pallets/vault-registry/src/types.rs

```diff
@@ -346,15 +346,6 @@ impl<T: Config> RichVault<T> {
 		self.decrease_issued(tokens)
 	}
 
-	fn update_stake(&self) -> DispatchResult {
```

Comment:
Why has this logic been removed?

Reply:
Without looking at this now in detail, does this mean that we changed the logic of the fee rewards and how they are distributed?

Reply:
Yes, the fee and native currencies are now distributed based on the amount of collateral in USD, not by the amount of tokens they have bridged. 
As an aside note, this is more aligned to what [interlay does](https://github.com/interlay/interbtc/blob/90a5c1b707c00dc9035c7ab438a7db3c8e651c19/crates/vault-registry/src/pool_manager.rs#L63) at the moment. With the difference we do not divide by the threshold.

Reply:
This change is fine as discussed yesterday.